### PR TITLE
[new release] universo and dedukti (2.7)

### PR DESCRIPTION
**Pull request by gabrielhdt**
A tool for Dedukti to play with universes

- Project page: <a href="https://github.com/Deducteam/Dedukti">https://github.com/Deducteam/Dedukti</a>

### CODE REVIEW

File: packages/universo/universo.2.7/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+authors: ["dedukti-dev@inria.fr"]
+license: "CECILL-B"
+homepage: "https://github.com/Deducteam/Dedukti"
+bug-reports: "https://github.com/Deducteam/Dedukti/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08"}
+  "z3"
+  "tezt" {with-test & >= "2.0"}
+]
```
Comment (mseri) [inline packages/universo/universo.2.7/opam:14]:
```suggestion
  "dedukti" {= version}
]
```

File: packages/dedukti/dedukti.2.7/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+authors: ["dedukti-dev@inria.fr"]
+license: "CECILL-B"
+homepage: "https://github.com/Deducteam/Dedukti"
+bug-reports: "https://github.com/Deducteam/Dedukti/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08"}
+  "cmdliner" {>= "1.1.0"}
+  "tezt" {with-test & >= "2.0"}
+  "menhir"
```
Comment (mseri) [inline packages/dedukti/dedukti.2.7/opam:14]:
```suggestion
  "menhir" {>= "20180523"}
```
For `--infer-write-query`

**Comment by gabrielhdt** [comment]
Hi, thank you for these edits, I realised there were some errors upstream, so it may be the case the archives cannot be retrieved anymore. I'll fix that up upstream, and get back to you, if it's OK.

**Comment by gabrielhdt** [comment]
Only revdeps for ocaml 4.14 fails (archsat) and linting fails. Is this OK or does it require changes on our side?

File: packages/dedukti/dedukti.2.7/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+bug-reports: "https://github.com/Deducteam/Dedukti/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08"}
+  "cmdliner" {>= "1.1.0"}
+  "tezt" {with-test & >= "2.0"}
+  "menhir" {>= "20180523"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/dedukti/dedukti.2.7/opam:17]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/universo/universo.2.7/opam [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+bug-reports: "https://github.com/Deducteam/Dedukti/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08"}
+  "z3" {>= "4.8.11"}
+  "tezt" {with-test & >= "2.0"}
+  "dedukti" {= version}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/universo/universo.2.7/opam:17]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
Just a minor thing to fix the lint, this can be done at the level of the opam-repository, you don't need to do anything for that. The other failure is independent of this, you can safely ignore it.

**Pull request merged**